README.md

```diff
@@ -1,72 +1,81 @@
 
- - The admins can add products and categories with the commands `/add_product` and `/add_category`. 
+- Type /show_categories to see products by category.
+- To search for something, you can use the command /search.
+- To buy something, you need to register a password. To do this, type /register.
+- The admins can add products and categories with the commands /add_product and /add_category.
 
-## screenshot
+## 🚸 Tutorial
 
 ### Register Command
 
-![Using the register command](/assets/register_command.gif)
+![Using the register command](./assets/register_command.gif)
 
 ### Seeing products
 
-![Seeing products](/assets/show_categories_command.gif)
+![Seeing products](./assets/show_categories_command.gif)
 
 ### Making a payment
 
-![Making a payment](/assets/payment.gif)
+![Making a payment](./assets/payment.gif)
 
 ### Searching for a product
 
-![Searching a product](/assets/search.gif)
+![Searching a product](./assets/search.gif)
 
 ### Changing the language
 
-![Changing the language used](/assets/changing_the_language.gif)
+![Changing the language used](./assets/changing_the_language.gif)
 
-## how to setup
+## ⚙️ How to set up ?
 
-First of all, see the dependencies in the requirements file or type `pip install -r requirements.txt` to install the dependencies automatically.
+First of all, see the dependencies in the requirements file or type pip install -r requirements.txt to install the dependencies automatically.
 
-The second process is to create a bot with the [Bot Father](https://core.telegram.org/bots#6-botfather) and get your bot token and to make test payments you should have a token from a payment provider, learn more about this in the [telegram payment page](https://core.telegram.org/bots/payments).
+The second process is to create a bot with the Bot Father and get your bot token. To make test payments, you should have a token from a payment provider. Learn more about this on the Telegram payment page.
 
-Now, put your tokens in the file `/telegram_ecommerce/utils/user_credentials.json`, in this file you can put the admin's usernames and the credentials of your MySQL database. Moreover, you can change some settings in the `consts.py` file, like the default language.
+Now, put your tokens in the file /telegram_ecommerce/utils/user_credentials.json. In this file, you can put the admin's usernames and the credentials of your MySQL database. Moreover, you can change some settings in the consts.py file, like the default language.
 
-After all, you can run the bot typing:
+After all, you can run the bot by typing:
 
 ```sh
  $ python bot.py
 ```
 
-## Disclaimer
+or
 
-This project it's just a demo, it's only have learning propose. To be ready for production, many features are need to be added. So I don't have the proposal to make it production-ready.
+```sh
+ $ bash run.sh
+```
 
+📌 **PS**: Docker is needed. Installation instruction [here.](https://docs.docker.com/engine/install/)
```

Comment:
Docker is only needed if the user want to run the bot inside a container. To turn it explicity, I suggest the following:

```suggestion
or, if you want to run the bot inside a container, use:

 $ bash run.sh

📌 **PS**: To run the command above Docker is needed. Installation instruction [here.](https://docs.docker.com/engine/install/)
```